src/content/blog/biome-v1-9.md

```diff
@@ -0,0 +1,245 @@
+summary: |
+  TODO
+authors:
+  - conaclos
+  - team
+pubDate: 2024-09-12
+coverImage:
+  lightSrc: "@/assets/blog/roadmap-2024/banner-light.png"
+  darkSrc: "@/assets/blog/roadmap-2024/banner-dark.png"
+  alt: The brand of the project. It says "Biome, toolchain of the web"
+socialImage: "@/assets/social-logo.png"
+---
+
+Today we’re excited to announce the release of Biome v1.9 and to celebrate the first anniversary of Biome 🎊
+
+Let's take a look back at the first year of Biome and then explore the new features in Biome 1.9.
+
+
+## One year of Biome
+
+We officially [announced Biome](/blog/annoucing-biome/) on 29 August 2023.
+From its inception, Biome has been a community-driven project.
+We have a governance and a solid base of contributors.
+
+In October 2023, one of the creators of Prettier launched [the Prettier challenge](https://console.algora.io/challenges/prettier) that rewarded any project written in Rust that passes at least 95% of the Prettier tests for JavaScript.
+We quickly organized ourselves to get there as soon as possible.
+By the end of November, we [surpassed this goal](/blog/biome-wins-prettier-challenge/) by passing 97% of the Prettier tests for JavaScript, as well as TypeScript, JSX and TSX!
+In the process, we identified several formatting issues in Prettier.
+This has also pushed contributions to Prettier to greatly improve its performance.
+This challenge was a win for the whole web ecosystem!
+
+Winning the challenge brought Biome to light.
+People have shown a lot of interest and Biome has been quickly adopted by many projects, including big ones such as [Ant Design](https://ant.design/), [Astro](https://astro.build/), [Sentry](https://sentry.io/), [daisyUI](https://daisyui.com/), [Refine](https://refine.dev/), [Discord](https://discord.com/), [Pulumi](https://www.pulumi.com/), [Label Studio](https://labelstud.io/), [Spicetify](https://spicetify.app/), [Apify](https://apify.com/), [Slint](https://slint.dev/), [Rspack](https://rspack.dev/), [FluidFrameowrk](https://fluidframework.com/), [and others](https://sourcegraph.com/search?q=file:biome.json&patternType=literal&sm=0).
+
+We surpassed 2.7 million monthly NPM downloads in August 2024.
+
+![Biome monthly NPM downloads](../../assets/blog/biome-v1-9/biome-monthly-npm-downloads.svg)
+
+We also gained sponsorship, notably [Shiguredo](https://shiguredo.jp/), [l2BEAT](https://l2beat.com/), [Phoenix Labs](https://www.phoenixlabs.dev/), [KANAME](https://kanamekey.com/), [Nanabit](https://nanabit.dev/), [Vital](https://vital.io/), [CodeRabbit](https://coderabbit.ai/), and [Forge42](https://forge42.dev/).
+These sponsorships are vital to the project.
+
+We also gained many new contributors.
+We started with a team of 5 core contributors, and we are now a team of 8 core contributors and 10 maintainers.
+
+In June 2024, Biome won the [JSNation's productivity booster Open Source Award](https://x.com/thejsnation/status/1805561129915060248).
```

Comment:
Really love this section full of highlights! Great way to reflect on our achievements :heart:

Reply:
I still expect some improvement. However, I think it is a good start :)